scripts/enforce_licence.py

```diff
@@ -239,9 +262,15 @@ def CheckLicenceForFile(file_name, licence, lang, diff, fix):
       # First line isn't a shebang, ignore it.
       f.seek(0, os.SEEK_SET)
       first_line = None
-  header = f.read(header_size)
+  # strip the trailing newline off as we don't actually want it
+  header = f.read(header_size).rstrip('\n')
+  file_name_line = f.readline()  
   f.close()
   if header == licence:
+    if lang != JS and file_name_line.rstrip('\n') != TransformLine(os.path.basename(file_name), lang):    
+      print "File %s does not have a filename line after the licence; found \"%s\" expected \"%s\"" % (
+          file_name, file_name_line.rstrip('\n'), TransformLine(os.path.basename(file_name), lang))
```

Comment:
put TransformLine(os.path.basename(file_name), lang) in an variable:

expected_line = TransformLine(os.path.basename(file_name), lang)